### PR TITLE
Navbar components

### PR DESCRIPTION
**Pull request by Witchery**
added responsive bootstrap navbar with routing to pages

### CODE REVIEW

**Review by filias — CHANGES_REQUESTED** [review]
Your editor somehow removes the newline at the end of the files. Please fix that. We do want a newline at the end of each file :)
It has to be something on your editor, some options for newlines at the end of the files.

File: frontend/src/index.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import Menu from './Menu';
```
Comment (filias) [inline frontend/src/index.js:4]: Why did you remove the Menu?

File: frontend/src/Menu.js [hunk 2]
```diff
@@ -1,20 +0,0 @@
-import React, {useState, useEffect} from 'react';
```
Comment (filias) [inline frontend/src/Menu.js:1]: Why did you delete this file?

File: frontend/src/index.js (hunk 1 above)
Comment (Witchery) [reply frontend/src/index.js:4]: because I replace it by App.js (there is the page divided to two parts (header, main) and used little bit different logic for navbar)

Comment (filias) [reply frontend/src/index.js:4]: but I cannot see the Menu from the api call anymore.

Comment (filias) [reply frontend/src/index.js:4]: Just to clarify. We had 3 api calls from the react frontend to our backend (flask or fastAPI). One was the menu, another the events and the other was the about.

Right now I stopped seeing those 3 calls in the code and in the frontend. Can you make sure that we continue having the same 3 calls and we show their results as it was before? Thanks :)

**Comment by Witchery** [comment]
> Your editor somehow removes the newline at the end of the files. Please fix that. We do want a newline at the end of each file :)

I think it's because of Windows. How can I fix that?

File: frontend/src/index.js (hunk 1 above)
Comment (filias) [reply frontend/src/index.js:4]: Basically when we add functionality we should not remove unrelated functionality. The about call is returning some description and scraping some info from the official pyladies website. We should keep it that way.

The menu might not be so important but again we should not remove functionality in one PR that is not aimed at doing that :) does that make sense?

Comment (Witchery) [reply frontend/src/index.js:4]: I have added back the events and scraping in react_hooks branch. Please check it.

Comment (Witchery) [reply frontend/src/index.js:4]: Now I understand why the about, events pages call to backend.  My branch didn't contain this files. Therefore I was confused.

Comment (filias) [reply frontend/src/index.js:4]: I see, you should always make a branch from the latest master :) we can review those things in the next meetup, I think it is useful for everyone :)

Comment (Witchery) [reply frontend/src/index.js:4]: > I see, you should always make a branch from the latest master :) we can review those things in the next meetup, I think it is useful for everyone :)

definitely but I thought that git branch <name> does it.

Comment (Witchery) [reply frontend/src/index.js:4]: git_branch

Comment (filias) [reply frontend/src/index.js:4]: I have no idea how you made your branch but if you say some files were missing then you probably did not have the latest master. Even if you created your branch before new commits went into master it is your responsibility to make sure your branch has every commit that is in master :)
